wikiclue/src/components/dailyEnd.svelte

```diff
@@ -1,44 +1,55 @@
 <script lang="ts">
-    import Overlay from "./overlay.svelte";
-	import { writable } from "svelte/store";
-    export let levelOpen = false;
+	import { goto } from '$app/navigation';
+	import Overlay from './overlay.svelte';
+	import { writable } from 'svelte/store';
+	export let levelOpen = false;
 
-    let statistics = {'Played' : 24, 'Win %': '85%', 'Current Streak': 3, 'Max Streak': 7};
-    let distribution = [4, 1, 3, 3, 4, 1];
-    let max = 4;
-    const isOverlayOpen = writable(false);
+	export let statistics = { Played: 24, 'Win %': '85%', 'Current Streak': 3, 'Max Streak': 7 };
+	export let distribution = [4, 1, 3, 3, 4, 1];
+	let max = 4;
+	const isOverlayOpen = writable(false);
 </script>
 
+<Overlay
```

Comment:
this file is all mangled with spacing, a linter would probably fix it tho

Reply:
This has been fixed, however it was not my file in the first place so maybe something we watch out for in the future